src/main/java/team4/footwithme/member/api/MemberApi.java

```diff
@@ -1,11 +1,33 @@
 package team4.footwithme.member.api;
 
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import team4.footwithme.global.api.ApiResponse;
+import team4.footwithme.member.api.request.JoinRequest;
+import team4.footwithme.member.api.request.LoginRequest;
+import team4.footwithme.member.service.MemberService;
+import team4.footwithme.member.service.response.MemberResponse;
 
 @RestController
 @RequiredArgsConstructor
-@RequestMapping
+@RequestMapping("/api/v1/members")
 public class MemberApi {
+
+    private final MemberService memberService;
+
+    @PostMapping("/join")
+    public ApiResponse<MemberResponse> join(@RequestBody @Valid JoinRequest request){
+        return ApiResponse.created(memberService.join(request.toServiceRequest()));
+    }
+
+    @PostMapping("/login")
+    public ApiResponse<?> login(@RequestBody @Valid LoginRequest request, HttpServletResponse response){
+        return ApiResponse.ok(memberService.login(request.toServiceRequest(), response));
+    }
```

Comment:
저는 와일드카드로 응답을 넘겨주는건 선호하지 않는 편이에요 만약 응답이 있다면 응답을 넣어주시고 없다면 Void와 같이 하면 좋을 것 같아요
```suggestion
    @PostMapping("/login")
    public ApiResponse<Void> login(@RequestBody @Valid LoginRequest request, HttpServletResponse response){
        return ApiResponse.ok(memberService.login(request.toServiceRequest(), response));
    }
```

아니면 뭔가 딱히 넣을 무언가가 없다라면 와일드카드를 써도되긴하지만... 전 안쓰는게 좋다고생각합니다..
sonarqube라는 Lint와 같은 코드품질을 측정해주는게 있는데 그 때 와일드카드를 전부 다 잡아내더라구요
수정해볼 수 있다면 수정하면 좋을 것 같아요

Reply:
아 이거 맨 처음 컨트롤러에 아직 이름을 안정하고 먼저 써놓고 나중에 바꾸는데 깜빡하고 안바꿨네요